src/styles/main.scss

```diff
@@ -0,0 +1,43 @@
+.clock {
+  display: flex;
+  justify-content: center;
+  width: 500px;
+  height: 500px;
+
+  &__minute {
+    position: absolute;
+    bottom: 50%;
+    height: 250px;
+    width: 8px;
+    background-color: blue;
+    animation: move 3600s linear infinite;
+    transform-origin: bottom;
+    border-radius: 8px;
+  }
+
+  &__second {
+    position: absolute;
+    bottom: 50%;
+    height: 150px;
+    width: 4px;
+    animation: move 60s linear infinite;
+    transform-origin: bottom;
+    background-color: red;
+    border-radius: 8px;
+  }
```

Comment:
There are some duplicated styles. Use extends, mixins or common classes to avoid it